package.json

```diff
@@ -13,8 +13,8 @@
     "prepare": "patch-package",
     "build:tsc": "tsc --outDir build",
     "build": "npm-run-all -s generate:api-models build:tsc",
-    "generate:api-models:public": "gen-api-models --api-spec https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/master/api/public_api_v1.yaml --no-strict --out-dir src/api",
-    "generate:api-models:admin": "gen-api-models --api-spec https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/master/api/admin_api.yaml --no-strict --out-dir src/api",
+    "generate:api-models:public": "rimraf generated/api && mkdir -p generated/api && gen-api-models --api-spec https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/master/api/public_api_v1.yaml --no-strict --out-dir generated/api",
```

Comment:
also here the URL should point to `v0.64.0` instead of `master` and since you're changing it already, the repository has been renamed from `digital-citizenship-functions` to `io-functions`